room.js

```diff
@@ -1,703 +0,0 @@
-        } else {
-            return console.log("You need to reach the hayloft first!")
-        }
-    },
-    takeHorseshoe() {
-        let horseshoe = this.inv.pop()
-        player.inv.push(horseshoe)
-        return console.log("The numbers 12345 are inscribed on the horseshoe")
-    }
-}
-
-
-
-// let objBox = []
-// player.inv.push(room1_cardboard_box, room1_box_key)
-
-//// FUCNTIONS /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-
-function check_stuff(inventory){
-    inventory.forEach(element => console.log(element.desc))
-    console.log('checked')
-    process.exit()
-}
-
-
-let room1 = {
-    inventory: [
-        'cabinet',
-        'writingDesk',
-        'paper'
-      ],
-     
-      north_description: "\nThere is a metal door with a keypad. A note on the side that says \'I hope you are familiar with Half-Life...\' ",
-      south_description: "\nThere is a wooden chest that is boarded up with not only a metal lock keeping it shut tight but wooden planks nailed into preventing access...",    // Player must remove planks first and THEN unlock the lock to get the note for the keypad
-      east_description: "\nYou see a writing desk against the east wall with a note written on top of it and a drawer slightly open",                                           // We want the player to open this box at any time to get the crowbar
-      west_description: "\nThere's a closed, cardboard box with a tear on the top",
-}
-
-let room2 = {
-    inventory: [
-        'pizza'
-    ],
-
-    north_description: "\n A sign is attatched to the front of the building: \"Caeser's Pizza Shop\". It appears the door to the building is locked, but there's plenty of customers inside. What gives!? ",
-    south_description: "\n There's a lingering key with a pizza keyring attatched to it on a nearby trashcan. Surely the owner wouldn't mind if you borrowed it?",   
-    east_description: "\n There's a busy street over here! Looks dangerous...",                                           
-    west_description: "\n Plenty of shops detailing other items you can buy later when this feature gets added!",
-
-    describe(direction){
-        switch(direction){
-            case 'north':
-                return this.north_description
-                break
-            
-            case 'south':
-                return this.south_description
-                break
-
-            case 'east':
-                return this.east_description
-                break
-
-            case 'west':
-                return this.west_description
-                break
-
-            default:
-                console.log("Error... Please enter a cardinal direction (north, south, east, west) as the argument")
-                break
-        }
-    }
-
-}
```

Comment:
These two rooms are similar enough that they could be condensed using a `class` then using a `let room = (..)` for each of them with their own descriptions.